lib/mongodb.js

```diff
@@ -751,6 +751,13 @@ function buildNearFilter(query, near) {
       },
     },
   };
+
+  var props = ['maxDistance', 'minDistance'];
+  props.forEach(function(p) {
+    if (near[p]) {
+      query.where[near.key]['near']['$' + p] = near[p];
```

Comment:
@seanbrookes you're right, near filter can set unit, I've already update this, make 'miles' as default unit, but I'm wondering if I should set 'meters' as default since it's default for mongo near query, I think people will intuitively use meter as default for mongo near query